Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM adoptopenjdk/openjdk11-openj9
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Always tag the version of an image explicitly](https://app.codacy.com/manual/NiccoMlt/Protelis-Web/pullRequest?prid=5040950)